src/index.js

```diff
@@ -52,10 +54,23 @@ function configureToMatchImageSnapshot({
     } = this;
     const chalk = new Chalk({ enabled: !noColors });
 
+    const retryTimes = global[Symbol.for('RETRY_TIMES')];
```

Comment:
Does Jest provide a better way of getting this information?

Reply:
This is how jest is pulling in the setting https://github.com/facebook/jest/blob/master/packages/jest-circus/src/run.js#L48